### PR TITLE
Auth and equipments

### PR DESCRIPTION
**Pull request by omersuve**
adding equipments to database

### CODE REVIEW

**Review by fileames — CHANGES_REQUESTED** [review]
I think we should not delete files to update but make changes on the same file.

**Comment by omersuve** [comment]
I had deleted the files trying to fit the directory structure. The structure of the main branch was changed and I should have instead merge the main branch, before pushing.
Thanks for the warning.

File: practice-app/practice-app/website/api/equipment.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        else:
+            new_equipment = Equipment(equipmentName=enteredEquipment, price=price)
+            
+            try:
+                db.session.add(new_equipment)
+                db.session.commit()
+                flash('Successfull', category='success')
+            except exc.NoReferenceError as e:
+                db.session.rollback()
+                return "User Not Registered", 400
```
Comment (fileames) [inline practice-app/practice-app/website/api/equipment.py:41]: I think there there is a minor mistake, "User not registered" is not related here.

File: practice-app/practice-app/website/api/equipment.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        tempEquipment = Equipment.query.filter_by(equipmentName=enteredEquipment).first()
+        if tempEquipment:
+            flash('Equipment already exists!.', category='error')
+        else:
+            new_equipment = Equipment(equipmentName=enteredEquipment, price=price)
+            
+            try:
+                db.session.add(new_equipment)
+                db.session.commit()
+                flash('Successfull', category='success')
```
Comment (fileames) [inline practice-app/practice-app/website/api/equipment.py:38]: flash is related to frontend, maybe returning with correct code would be better here

**Comment by fileames** [comment]
I realized there is another practice-app folder inside practice-app folder and some files are repeating. You can check the correct structure on master branch and select the correct folders.

**Comment by omersuve** [comment]
> I realized there is another practice-app folder inside practice-app folder and some files are repeating. You can check the correct structure on master branch and select the correct folders.

Unrelated files and directories are deleted, thanks.

File: practice-app/practice-app/website/api/equipment.py (hunk 1 above)
Comment (omersuve) [reply practice-app/practice-app/website/api/equipment.py]: Thank you, I have fixed it!

**Review by eye-78 — CHANGES_REQUESTED** [review]
i didnt understand bin files
i didnt understand new models.py file

please edit views.py in an appropriate way

File: practice-app/website/test/equipment_test.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import unittest
+import requests, json
+
+class TestEquipmentAPI(unittest.TestCase):
```
Comment (fileames) [inline practice-app/website/test/equipment_test.py:4]: As discussed on the team meeting, unit test should not make requests to endpoints. Please update unit tests.

**Review by fileames — CHANGES_REQUESTED** [review]

File: practice-app/website/test/equipment_test.py (hunk 3 above)
Comment (omersuve) [reply practice-app/website/test/equipment_test.py]: Done, thanks!

**Comment by omersuve** [comment]
Please merge my branch.

**Comment by berkatil** [comment]
thanks!

**Pull request merged**